src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -33,38 +35,28 @@ private void readObject(java.io.ObjectInputStream in)
     private static final String pluginPackageName = "org.broadinstitute.hellbender.engine.filters";
     private static final Class<?> pluginBaseClass = org.broadinstitute.hellbender.engine.filters.ReadFilter.class;
 
-    @Argument(fullName = StandardArgumentDefinitions.READ_FILTER_LONG_NAME,
-            shortName = StandardArgumentDefinitions.READ_FILTER_SHORT_NAME,
-            doc="Read filters to be applied before analysis", optional=true)
-    public final List<String> userReadFilterNames = new ArrayList<>(); // preserve order
-
-    @Argument(fullName = StandardArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME,
-            shortName = StandardArgumentDefinitions.DISABLE_READ_FILTER_SHORT_NAME,
-            doc="Read filters to be disabled before analysis", optional=true)
-    public final Set<String> disableFilters = new HashSet<>();
-
-    @Argument(fullName = "disableAllReadFilters",
-            shortName = "disableAllReadFilters",
-            doc = "Disable all read filters", common = false, optional = true)
-    public boolean disableAllReadFilters = false;
+    @ArgumentCollection
+    protected final GATKReadFilterPluginArgumentCollection args;
 
     // Map of read filter (simple) class names to the corresponding discovered plugin instance
-    private Map<String, ReadFilter> readFilters = new HashMap<>();
+    protected Map<String, ReadFilter> readFilters = new HashMap<>();
 
     // List of default filters in the order they were specified by the tool
-    private List<String> toolDefaultReadFilterNamesInOrder = new ArrayList<>();
+    protected final List<String> toolDefaultReadFilterNamesInOrder = new ArrayList<>();
 
     // Map of read filter (simple) class names to the corresponding default plugin instance
-    private Map<String, ReadFilter> toolDefaultReadFilters = new HashMap<>();
+    protected final Map<String, ReadFilter> toolDefaultReadFilters = new HashMap<>();
 
     // Set of dependent args for which we've seen values (requires predecessor)
-    private Set<String> requiredPredecessors = new HashSet<>();
+    protected final Set<String> requiredPredecessors = new HashSet<>();
 
     /**
+     * @param args               Argument collection for the pluging.
      * @param toolDefaultFilters Default filters that may be supplied with arguments
      *                           on the command line. May be null.
      */
```

Comment:
I know you didn't write this comment (I think I did), but can you change it so say "Filters that are enabled by default for this tool. May be null."